Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -8287,7 +8287,7 @@ RefPtr<CSSValue> consumeCounterStyleAdditiveSymbols(CSSParserTokenRange& range,
             return nullptr;
         lastWeight = weight;
 
-        return CSSValueList::createSpaceSeparated(integer.releaseNonNull(), symbol.releaseNonNull());
+        return CSSValuePair::create(integer.releaseNonNull(), symbol.releaseNonNull());
```

Comment:
Switching from a list to a pair here requires changing the code that interprets this value to access it as a pair rather than a list. I don’t see that change here. I think that’s the reason for the crashes.

Reply:
The code that actually uses the parsed counter-style shouldn't be reachable by now, since the fragment wiring it to RenderListMarker is in a separated PR.